constraints/Cargo.toml

```diff
@@ -24,5 +24,10 @@ lazy_static = "1.4.0"
 serde_json = { version = "1.0.91", features = ["preserve_order"] }
 
 [features]
-default = []
+default = ["serde"]
 serde = ["dep:serde", "indexmap/serde"]
+
+[[example]]
```

Comment:
what about `macros` and `native` examples? should we add them here too?

Reply:
The reason for only the `json` example being listed is that it requires specific features (`serde`).